pkg/cmd/issue/edit/edit.go

```diff
@@ -49,7 +49,7 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 	var removeMilestone bool
 
 	cmd := &cobra.Command{
-		Use:   "edit {<numbers> | <urls>}",
+		Use:   "edit [<numbers> | <urls>]",
```

Comment:
So, I think this needs to change a little bit more because I first thought there were typos here `numbers`/`urls` but this commands actually does take multiple.  However the usage statement doesn't really communicate it in POSIX style